src/Coordination/FourLetterCommand.cpp

```diff
@@ -472,4 +478,24 @@ String ApiVersionCommand::run()
     return toString(static_cast<uint8_t>(Coordination::current_keeper_api_version));
 }
 
+String CreateSnapshotCommand::run()
+{
+    auto log_index = keeper_dispatcher.createSnapshot();
+    return log_index > 0 ? "Snapshot creation scheduled with last committed log index " + std::to_string(log_index) + "."
```

Comment:
Why not just return the index on success nad failure message otherwise?
That's how we handle, for example, `mntr`.